boards.txt

```diff
@@ -4337,6 +4337,171 @@ sparkfun_lora_gateway_1-channel.menu.DebugLevel.verbose.build.code_debug=5
 
 ##############################################################
 
+sparkfun_esp32_iot_redboard.name=SparkFun ESP32 IoT RedBoard
+
+sparkfun_esp32_iot_redboard.bootloader.tool=esptool_py
+sparkfun_esp32_iot_redboard.bootloader.tool.default=esptool_py
+
+sparkfun_esp32_iot_redboard.upload.tool=esptool_py
+sparkfun_esp32_iot_redboard.upload.tool.default=esptool_py
+sparkfun_esp32_iot_redboard.upload.tool.network=esp_ota
+
+sparkfun_esp32_iot_redboard.upload.maximum_size=1310720
+sparkfun_esp32_iot_redboard.upload.maximum_data_size=327680
+sparkfun_esp32_iot_redboard.upload.flags=
+sparkfun_esp32_iot_redboard.upload.extra_flags=
+
+sparkfun_esp32_iot_redboard.serial.disableDTR=true
+sparkfun_esp32_iot_redboard.serial.disableRTS=true
+
+sparkfun_esp32_iot_redboard.build.tarch=xtensa
+sparkfun_esp32_iot_redboard.build.bootloader_addr=0x1000
+sparkfun_esp32_iot_redboard.build.target=esp32
+sparkfun_esp32_iot_redboard.build.mcu=esp32
+sparkfun_esp32_iot_redboard.build.core=esp32
+sparkfun_esp32_iot_redboard.build.variant=sparkfun_esp32_iot_redboard
+sparkfun_esp32_iot_redboard.build.board=ESP32_DEV
```

Comment:
this value should be updated to uniquely identify the board.

Reply:
Whoops, thanks for catching that!